src/components/DetailComponent/DetailComponent.js

```diff
@@ -0,0 +1,253 @@
+import { Container, Grid } from "@material-ui/core";
+import React, { Fragment, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import { IMAGE_PATH } from "../../store/constants";
+import Typography from "@mui/material/Typography";
+import Loader from "../../shared/loader";
+import { detailStyle } from "../../styles/appStyles";
+
+export default function DetailComponent({
+  detailData,
+  detailReq,
+  detailLoader,
+  reviews,
+  error,
+  match,
+}) {
+  const classes = detailStyle();
+  const routesData = match.params;
+  useEffect(() => {
+    window.scrollTo(0, 0);
+    detailReq(routesData);
+    console.log(reviews);
+  }, []);
+
+  const convertToString = (data) => {
+    let str = "";
+    data?.forEach((recipe) => {
+      str =
+        recipe?.name?.concat(" ", str) ||
+        recipe?.original_title?.concat(" ", str);
+    });
+    return str;
+  };
+  return (
+    <Fragment>
+      {detailLoader === true ? (
+        <Container style={{ height: "100vh" }}>
+          <Loader />
+        </Container>
+      ) : (
+        <Fragment>
+          <Typography gutterBottom variant="h3" component="div">
+            {detailData.title ?? detailData.original_name ?? detailData.name}
+          </Typography>
+          <Grid
+            container
+            style={{ marginTop: "3rem", padding: "0 2rem 0 2rem" }}
+          >
+            <Grid item xl={6} lg={6} md={6} sm={6} xs={12}>
+              {routesData.component === "person" && (
+                <img
+                  className={classes.imgStyle}
+                  src={IMAGE_PATH + detailData?.profile_path}
+                />
+              )}
+              {routesData.component === "tv" && (
+                <img
+                  className={classes.imgStyle}
+                  src={IMAGE_PATH + detailData?.poster_path}
+                />
+              )}
+              {routesData.component === "movie" && (
+                <img
+                  className={classes.imgStyle}
+                  src={IMAGE_PATH + detailData?.poster_path}
+                />
+              )}
+            </Grid>
+            {routesData.component !== "person" ? (
+              <Grid item xl={6} lg={6} md={6} sm={6} xs={12}>
+                <Typography
+                  gutterBottom
+                  variant="h5"
+                  className={classes.headingText}
+                  component="div"
```

Comment:
Try not to use it, unless necessary.

Reply:
Removed the component ="div" prop from typography